src/components/RichTextEditor/utils/exampleValues.ts

```diff
@@ -18,13 +18,16 @@ import { ELEMENT_CHECK_ITEM } from '../Plugins/CheckboxListPlugin/id';
 import { MentionComboboxItem, MentionableCategory } from '../Plugins/MentionPlugin/types';
 import { TextStyles } from '../Plugins/TextStylePlugin/TextStyles';
 import { DesignTokens } from '../types';
+import { mixedMarkdown } from '../serializer/markdown/__tests__/fixtures';
```

Comment:
Mixed feeling about importing tests data in here
Is it used only in stories? If yes, why not importing it there?

Reply:
No, it is not. it is used in test. So to not create a duplicate and if this works in test it will work in RTE :)